docs/sources/git.md

```diff
@@ -1,29 +1,86 @@
-# Git based bundles
+# Git source
 
 ## Summary
 
-A Bundle can reference content in a remote git repository instead of a remote container image by using the
-`git` source type in the Bundle manifest. This enables one to easily source content from a git provider like GitHub and
-make it available within the cluster.
+The git source provides the contents in a git repository as the source of the bundle.  The `source.type` for the git source is `git`.
+When creating a git source, a reference to a particular commit, tag, or branch must be provided in addition to the URL of the
+repository and an optional directory in the repository. It is expected that a proper format of bundle content is present
+in the particular commit/tag/branch at the directory specified.
 
-For example, for a plain+v0 bundle, the git repository backing the bundle, or "bundle git repo", must have a certain
-structure in order to produce a valid Bundle that works with the plain provisioner. It should have a manifests directory
-where the Kubernetes manifests are stored -- this directory is used when copying manifests onto the cluster. By default,
-the `manifests` directory is assumed to be at the root level, but a different bundle root can be specified
-via `spec.source.git.directory`.
+## Examples
+
+Referencing a git repository by a commit:
```

Comment:
Can we make these be `###` tags to make the document flow a bit better? Applies to all subsequently similar lines.

```suggestion
### Referencing a git repository by a commit
```